pkg/models/registry.go

```diff
@@ -38,210 +37,116 @@ var (
 // LocalRegistry defines model registry for managing registered local models. Local models are locally compiled into
 // the program binary and hence some additional information in compare to remote models, i.e. go type.
 type LocalRegistry struct {
```

Comment:
LocalRegistry was named "local" to differentiate from RemoteRegistry, but that doesn't exist anymore. So it is little bit confusing why "local" (if i didn't know story behind it).  Renaming it to Registry clash with the interface Registry handling all previous instances of Local/RemoteRegistry. But i guess we can make it private by lower capital and its usage replace with Register interface.